### PR TITLE
Tethyss4 upgrade

### PR DESCRIPTION
**Pull request by EricJJones**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (swainn) [inline .vscode/settings.json]: Remove this file. Use this command to remove it from the repository, but keep it locally:

```
git rm --cached .vscode/settings.json
```

Then add it to the .gitignore file so it doesn't get committed again in the future.

File: tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalFoldersAndFilesExplorer.html [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
                 <h4 class="modal-title">Catalog Files and Folders</h4>
+                <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
             </div>
             <div id="loading-for-modalFoldersAndFilesExplorer" class="loading-div" style="display: none">
                 <img class="loading-gif" src="{% static 'metdataexplorer/images/loading.gif' %}"/>
             </div>
             <div id="modal-body-for-modalFoldersAndFilesExplorer" class="modal-body">
-              <div id="up-a-level-btn" class="file">
-                <span class="file-folder-icon glyphicon glyphicon-menu-up"></span>
+              <div id="up-a-level-btn" class="file" onclick="">
```
Comment (swainn) [inline tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalFoldersAndFilesExplorer.html]: Is this `onclick=""` needed? If not, remove it.

File: tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalInformationAboutGroup.html [hunk 3]
```diff
@@ -2,7 +2,12 @@
     <div class="modal-dialog">
         <div class="modal-content myModal">
             <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal">&times;</button>
+                <h4>Information about Catalog</h4>
+                <!--<div id="pop-up_description2">
+                </div>-->
```
Comment (swainn) [inline tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalInformationAboutGroup.html]: What is going on here?

File: tethysapp/metdataexplorer/workspaces/app_workspace/.dodsrc [hunk 4]
```diff
@@ -0,0 +1,2 @@
+HTTP.COOKIEJAR=/home/tethys/tethysdev/tethysapp-metdataexplorer/tethysapp/metdataexplorer/workspaces/app_workspace/.urs_cookies
```
Comment (swainn) [inline tethysapp/metdataexplorer/workspaces/app_workspace/.dodsrc]: The files in `app_workspace` folder should be removed from the repository:

```
git rm --cached tethysapp/metdataexplorer/workspaces/app_workspace/*
```

**Review by swainn — CHANGES_REQUESTED** [review]
@EricJJones see comments.

File: tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalInformationAboutGroup.html (hunk 3 above)
Comment (EricJJones) [reply tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalInformationAboutGroup.html]: Oh that was working out how to get the close button at the correct place and left a trial item in a comment.

File: tethysapp/metdataexplorer/workspaces/app_workspace/.dodsrc (hunk 4 above)
Comment (swainn) [reply tethysapp/metdataexplorer/workspaces/app_workspace/.dodsrc]: These files got added again.

**Review by swainn — APPROVED** [review]

**Review by swainn — APPROVED** [review]

**Review by jenochjones — APPROVED** [review]

**Comment by swainn** [comment]
@jenochjones it looks like we have some conflicts that need to be resolved. Have you made changes to the master branch recently?

**Pull request merged**